### PR TITLE
docs: adding note about middlewares and slices

### PR DESCRIPTION
**Pull request by Cadienvan**
## Related Issues or Discussions

Fixes #2376.

## Summary

As mentioned, I started adding a note to discuss with @dbritto-dev about using middlewares in slices.

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Comment by Cadienvan** [comment]
@dbritto-dev let me know which wording you'd prefer to use, I started by applying a simple phrase, feel free to update it!

**Comment by dbritto-dev** [comment]
@Cadienvan LGTM

**Review by dai-shi — COMMENTED** [review]
👍

File: docs/guides/slices-pattern.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ export const useBoundStore = create(
 )
 ```
 
+Please, keep in mind you should apply middlewares only in the combined store and avoid applying them inside individual slices to prevent issues.
```
Comment (sewera) [inline docs/guides/slices-pattern.md:116]:
```suggestion
Please keep in mind you should only apply middlewares in the combined store. Applying them inside individual slices can lead to unexpected issues.
```

**Review by sewera — CHANGES_REQUESTED** [review]

**Comment by Cadienvan** [comment]
@sewera I applied your suggestionm!

**Review by dbritto-dev — APPROVED** [review]

**Pull request merged**